### PR TITLE
[sdks] Extract versions into a dedicated file

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

File: sdks/builds/ios.mk [hunk 1]
```diff
@@ -32,18 +32,20 @@ _ios_LDFLAGS=
 #  $(3): arch (arm or aarch64)
 #
 # Flags:
-#  ios_$(1)_AC_VARS
-#  ios_$(1)_CFLAGS
-#  ios_$(1)_CPPFLAGS
-#  ios_$(1)_CXXFLAGS
-#  ios_$(1)_LDFLAGS
-#  ios_$(1)_BITCODE_MARKER
+#  ios_$(_$(1)_normalized)_AC_VARS
```
Comment (vargaz) [inline sdks/builds/ios.mk]: Whats the point of these changes ? Why do we have to make these variables uppercase ?

Comment (luhenry) [reply sdks/builds/ios.mk]: I pushed the wrong version :)

**Comment by luhenry** [comment]
@monojenkins build Linux AArch64

**Comment by luhenry** [comment]
@monojenkins build Linux x64 FullAOT

File: sdks/builds/ios.mk [hunk 2]
```diff
@@ -11,19 +11,19 @@
 
 PLATFORM_BIN=$(XCODE_DIR)/Toolchains/XcodeDefault.xctoolchain/usr/bin
 
-_ios_CFLAGS= \
+ios_CFLAGS= \
```
Comment (vargaz) [inline sdks/builds/ios.mk]: These start with an underscore because they are used internally inside the template macros.

**Comment by luhenry** [comment]
@monojenkins build Linux AArch64

File: sdks/builds/ios.mk (hunk 2 above)
Comment (luhenry) [reply sdks/builds/ios.mk]: They are not inside the template, that's why I removed the `_`, but feel free to revert it back, I don't have a strong opinion about it

**Pull request merged**